scout/server/blueprints/clinvar/views.py

```diff
@@ -42,8 +42,13 @@ def clinvar_submission_status(submission_id):
     """Sends a request to ClinVar to retrieve and display the status of a submission."""
 
     # flash a message with current submission status for a ClinVar submission
+    clinvar_resp_status = dict(
```

Comment:
Nice breaking it out! Why the cast to dict - some missing value or occasional odd return type? Maybe the type hint on `json_submission_status` needs a refresher as well based on that?